src/Form/Form.js

```diff
@@ -53,9 +53,6 @@ class Form {
         this.addInput(input)
         this.tooltip = null
         this.activeInput = null
-        // TODO: try to filter down to only submit buttons?
```

Comment:
Moved to a getter to avoid problems when pages destroy and re-create the button (Microsoft).